tutorials/releases.md

```diff
@@ -0,0 +1,311 @@
+![CRI-O logo](../logo/crio-logo.svg)
+
+Welcome to the releases page! Enjoy CRI-O!
+
+- [1.18 (latest stable)](#118--latest-stable-)
+  * [1.18.0](#1180)
+    + [CentOS 8](#centos-8)
+  * [CentOS 8 Stream](#centos-8-stream)
+    + [Debian Unstable](#debian-unstable)
+    + [Debian Testing](#debian-testing)
+    + [Fedora 31 or later](#fedora-31-or-later)
+    + [Ubuntu 20.04](#ubuntu-2004)
+- [1.17](#117)
+  * [1.17.4](#1174)
+    + [CentOS 7](#centos-7)
+    + [CentOS 8](#centos-8-1)
+    + [CentOS 8 Stream](#centos-8-stream-1)
+    + [Debian Unstable](#debian-unstable-1)
+    + [Debian Testing](#debian-testing-1)
+    + [Fedora](#fedora)
+    + [Ubuntu 18.04](#ubuntu-1804)
+    + [Ubuntu 19.04](#ubuntu-1904)
+    + [Ubuntu 19.10](#ubuntu-1910)
+    + [Ubuntu 20.04](#ubuntu-2004-1)
+- [1.16](#116)
+  * [1.16.6](#1166)
+    + [CentOS 7](#centos-7-1)
+    + [CentOS 8](#centos-8-2)
+    + [CentOS 8 Stream](#centos-8-stream-2)
+    + [Debian Unstable](#debian-unstable-2)
+    + [Debian Testing](#debian-testing-2)
+    + [Fedora](#fedora-1)
+    + [Ubuntu 18.04](#ubuntu-1804-1)
+    + [Ubuntu 19.04](#ubuntu-1904-1)
+    + [Ubuntu 19.10](#ubuntu-1910-1)
+    + [Ubuntu 20.04](#ubuntu-2004-2)
+- [1.15 (deprecated)](#115--deprecated-)
+  * [1.15.4](#1154)
+    + [CentOS 7](#centos-7-2)
+    + [Debian Unstable](#debian-unstable-3)
+    + [Debian Testing](#debian-testing-3)
+    + [Fedora](#fedora-2)
+
+
+# 1.18 (latest stable)
+**Notes**:
+  - CRI-O 1.18.0 or higher version **requires golang 1.13 or higher**
+
+## 1.18.0
+### CentOS 8
+For CentOS 8 run the following as root:
+```
+cd /etc/yum.repos.d/
+wget https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:1.18:1.18.0/CentOS_8/devel:kubic:libcontainers:stable:cri-o:1.18:1.18.0.repo
+yum install cri-o
+```
+## CentOS 8 Stream
+For CentOS 8 Stream run the following as root:
+```
+cd /etc/yum.repos.d/
+wget https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:1.18:1.18.0/CentOS_8_Stream/devel:kubic:libcontainers:stable:cri-o:1.18:1.18.0.repo
+yum install cri-o
+```
+### Debian Unstable
+For Debian Unstable run the following as root:
+Keep in mind that the owner of the key may distribute updates, packages and repositories that your system will trust (more information).
```

Comment:
I'd change "key" to "Release.key" just to be clear which key you're talking about.  Also, you have "(more information)" which at first led me to believe it would be a link to more info on release keys and how they work.   However it seems to mean the following example.  Personally I'd just drop the parens entirely, or if you think it's valuable, change it to "(more information on Release.keys)" and make the "Release.keys" be a link to a page with more info on them.  Same comment through out.

Reply:
Thanks @TomSweeneyRedHat. I just copy/paste the message from OBS system and the link for "(more information)" was not included.